feat:add new information

kevinhuang09 opened the pull request:
add some button in HomeView.vue

kevinhuang09 committed:
feat:add new information

add some button in HomeView.vue

marvelcn015 commented:
Dear Kevin,

Thank you for submitting your first Pull Request. I appreciate your initiative and quick turnaround. After reviewing your submission, I would like to provide some constructive feedback to help improve the code quality and architecture.

1. I noticed that the ESLint validation has not passed. Before submitting any commits, please ensure you run the following commands to maintain our code standards:
```
pnpm lint
pnpm format
```

2. Regarding the navigation bar implementation, I recommend refactoring it into a **reusable component**. Since this element appears across multiple pages, componentization would improve maintainability and ensure consistency throughout the application.

3. I also observed that the link implementations in the `<a>` tags require attention. As you've utilized code segments from the template, the links are currently non-functional. I encourage you to explore and implement **Vue Router** concepts, particularly focusing on the `RouterLink` component, which is essential for client-side navigation in Vue applications.

Your swift contribution to the project is commendable, and I look forward to reviewing your refinements. Please don't hesitate to reach out if you need any clarification or assistance.

Best regards,
Aaron Yeh

kevinhuang09 committed:
feat:format and modification

I try to use prettier to verify format , and revise the <a> in the HomeView.vue
